modules/issue_tracker/jsx/index.js

```diff
@@ -4,12 +4,12 @@ import IssueForm from './IssueForm';
  * Render IssueForm on page load
  */
 $(function() {
-  const args = QueryString.get();
+  var id = location.href.split("/issue/")[1];
```

Comment:
const instead fo var

Reply:
@HenriRabalais does that looks like it makes sense to you Mr issue tracker guy ?

Reply:
@ridz1208 var isn't supposed to be used anymore. The two options are either const or let and with const being what should be used in this case like @kongtiaowang suggested.